src/test/java/org/apache/beam/sdk/io/gcp/healthcare/WebPathParserTest.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package org.apache.beam.sdk.io.gcp.healthcare;
+
+import java.io.IOException;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Testing the Healthcare API webpath parser.
+ */
+@RunWith(JUnit4.class)
+public class WebPathParserTest {
+
+    @Test
+    public void test_parsedAllElements() throws IOException {
+        String webpathStr = "projects/foo/location/earth/datasets/bar/dicomStores/fee/dicomWeb/studies/" +
```

Comment:
considering you are splitting this anyway wanna split it as <100chars in one line and the rest on the next?